applications/StructuralMechanicsApplication/custom_advanced_constitutive/generic_small_strain_high_cycle_fatigue_law.h

```diff
@@ -7,7 +7,7 @@
 //                   license: structural_mechanics_application/license.txt
 //
 //  Main authors:    Sergio Jimenez/Alejandro Cornejo/Lucia Barbu
-//  Collaborator:
+//  Collaborator:    
```

Comment:
can you please configure your IDE to automatically delete trailing whitespaces when saving a file?
this way many changes won't show up